frontend/public/components/storage/create-pvc.tsx

```diff
@@ -64,7 +82,11 @@ export class CreatePVCForm extends React.Component<CreatePVCFormProps, CreatePVC
   };
 
   handleStorageClass = storageClass => {
-    this.setState({ storageClass: _.get(storageClass, 'metadata.name'), storageClassAccessMode: _.get(storageClass, ['metadata', 'annotations', 'storage.alpha.openshift.io/access-mode'])}, this.onChange);
+    //if the provisioner is unknown or no storage class selected, user should be able to set any access mode
+    const modes = storageClass && storageClass.provisioner && provisionerAccessModeMapping[storageClass.provisioner] ? provisionerAccessModeMapping[storageClass.provisioner] : ['ReadWriteOnce', 'ReadWriteMany', 'ReadOnlyMany'];
+
+    //setting accessMode to default with the change to Storage Class selection
+    this.setState({ storageClass: _.get(storageClass, 'metadata.name'), storageClassAccessModes: modes, accessMode: 'ReadWriteOnce'}, this.onChange);
```

Comment:
It does sounds like we are deprecating storage.alpha.openshift.io/access-mode because it is not documented and no one uses it except console (great ideas that never catch on).  In this case, this change is okay.